### ISSUE TITLE
PaletteSlice not visible to circup as valid library

### ISSUE THREAD

**Issue by CedarGroveStudios** [adafruit/CircuitPython_Community_Bundle#127]
Just noticed that the submodule and file path for the recently added PaletteSlice helper library are in camelcase rather than all lowercase. What's the process to remove the new library and resubmit correctly -- or can it be fixed in place?

Thanks

**Comment by tannewt** [comment]
I think you can use `git mv` to change the folder name. `.gitmodules` may need to be updated along with it. (Hopefully `git mv` will do it automatically.)

**Comment by CedarGroveStudios** [comment]
Thank you!
`git mv` automatically updated the path in `.gitmodules` but left the submodule definition header with the camelcase version. I manually edited the header line and submitted a PR.

before `git mv`:
```
[submodule "libraries/helpers/PaletteSlice"]
	path = libraries/helpers/PaletteSlice
	url = https://github.com/CedarGroveStudios/CircuitPython_PaletteSlice.git
```

after `git mv`:
```
[submodule "libraries/helpers/PaletteSlice"]
	path = libraries/helpers/paletteslice
	url = https://github.com/CedarGroveStudios/CircuitPython_PaletteSlice.git
```

after editing `.gitmodules`:
```
[submodule "libraries/helpers/paletteslice"]
	path = libraries/helpers/paletteslice
	url = https://github.com/CedarGroveStudios/CircuitPython_PaletteSlice.git
```

Would there ever be an instance where the submodule definition and path wouldn't be all lower-case? Should a check or correction be automated in the CI, perhaps?

I appreciate your help, Scott!

**Comment by tekktrik** [comment]
Resolved via #129

**Comment by tannewt** [comment]
I don't think the capitalization actually matters. I wouldn't worry about it.
